source/Calamari/Integration/FileSystem/CalamariPhysicalFileSystem.cs

```diff
@@ -114,7 +117,7 @@ public void DeleteFile(string path, FailureOptions options)
 
         public void DeleteDirectory(string path)
         {
-            Directory.Delete(path, true);
+            Directory.Delete(path);
```

Comment:
The second parameter is recurse=true, why is it no longer required? I guess nothing did depend on it being recursive

I see the usage of does do `fileSystem.PurgeDirectory` first:
https://github.com/OctopusDeploy/Calamari/blob/2b45299e50393cb6153ea9b3ae159f7ed7c1d557/source/Calamari/Deployment/Retention/RetentionPolicy.cs#L85

the rest of the calls to `DeleteDirectory` are in tests.